src/api/menu/components/SuperMenu.vue

```diff
@@ -88,6 +93,12 @@ export default {
       hoveredItem: {}
     };
   },
+  computed: {
+    getLabel() {
```

Comment:
this is super nitpicky but `get` to me implies a method call

Reply:
Makes sense! I changed this to `optionsLabel`.